src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -177,7 +183,7 @@ public Uri SourceUri
                         Strings.DefaultSymbolServer));
                 }
 
-                await PushPackage(symbolPackagePath, source, apiKey, noServiceEndpoint, requestTimeout, log, token, isSnupkgPush: isSymbolEndpointSnupkgCapable);
+                await PushPackage(symbolPackagePath, source, apiKey, noServiceEndpoint, false, false, requestTimeout, log, token, isSnupkgPush: isSymbolEndpointSnupkgCapable);
```

Comment:
As other comments, I think it would be good to have something more understandable than "false, false".